src/util/hass-util.html

```diff
@@ -63,12 +65,23 @@
   Object.keys(attributes).forEach(function (key) {
     customEl[key] = attributes[key];
   });
+  customEl.isVisible = true;
```

Comment:
isVisible is not part of dynamicContentUpdater. dynamicContentUpdater only dynamically updates content, it doesn't mean that the content will be visible.